platform/pddf/i2c/modules/xcvr/pddf_xcvr_module.c

```diff
@@ -0,0 +1,291 @@
+/*
+ * Copyright 2019 Broadcom. All rights reserved.
+ * The term “Broadcom” refers to Broadcom Inc. and/or its subsidiaries.
```

Comment:
need gpl license

Reply:
Done

Reply:
taken care in 
@fk410167
1) GPL license disclaimer inclusion 2) Removal of commented code, as … …
8e8cf2f